front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -76,6 +95,20 @@ import { useEventSource } from "@app/hooks/useEventSource";
 import { useSubmitFunction } from "@app/lib/client/utils";
 import { useAgentConfigurationLastAuthor } from "@app/lib/swr/assistants";
 
+const typeIcons = {
```

Comment:
Duplicating from sparkle for now as the `Citation` component will be deprecated once this PR is shipped